src/Service/Logger.service.js

```diff
@@ -201,6 +201,25 @@ export default class LoggerService extends DependencyAwareClass {
     this.logger.log('info', this.processMessage(message));
   }
 
+  /**
+   * Logs an error, using `LoggerService.error`
+   * or `LoggerService.info` based on
+   * `process.env.LOGGER_SOFT_WARNING`.
+   *
+   * Please note that `LoggerService.error` and `LoggerService.info`
+   * have different signatures. The function uses the shared argument
+   * instead of introducing ambiguity.
+   *
+   * @param error
+   */
+  warning(error) {
+    if (process.env.LOGGER_SOFT_WARNING) {
```

Comment:
Guessing if we set `LOGGER_SOFT_WARNING=false` in the .env or wherever, that would be truthy?

Is it worth requiring it to to equal `'true'` or something, so no one will ever be tripped up by this? Maybe not worth worrying about?

Reply:
Good catch. We can check `Boolean(process.env.LOGGER_SOFT_WARNING)`, this would work with `true`, `false`, `1` and `0`

Reply:
`Boolean("0") === true`, so I was wrong in that

Reply:
Sounds good!

Reply:
how about `["true", "1"].includes(process.env.LOGGER_SOFT_WARNING)`?

Reply:
👍

Reply:
56bb4ee68350fb0cd6f09ab4650cbcd48fa3a50f